docs/src/components/language-selector.js

```diff
@@ -83,7 +83,6 @@ const Languages = props => (
   <Context.Consumer>
     {({ current, changeLanguage }) => (
       <Fragment>
-        <H3>Select language</H3>
```

Comment:
@andrewoh531 any reason you removed this? Probably an accessible heading. Happy to discuss whether we need or if we should make it invisible, but maybe let's do in a later PR.

Reply:
I've added it back in. I initially removed it as there were a lot of things going on on the page.